loops/cpp/code.cpp

```diff
@@ -0,0 +1,24 @@
+#include <string>
+#include <random>
+#include <array>
+#include <print>
+#include <cstdint>
+#include <ranges>
+
+int main (int argc, char* argv[])
+{
+    int const mod = std::stoi(argv[1]);                       // Get an input number from the command line
+    std::random_device rd{};
+    std::mt19937 engine{rd()};
+    std::uniform_int_distribution<int> dist{0, 9999};         // <random> library setup
+
+    int const picked_number = dist(engine);                   // Get a random number 0 <= r < 10k
+    std::array<std::int32_t, 10000> array{};                  // Array of 10k elements initialized to 0
+    for (int const i : std::ranges::iota_view{0, 10000})      // 10k outer loop iterations
+    {
+        for (int const j : std::ranges::iota_view{0, 100000}) // 100k inner loop iterations, per outer loop iteration
```

Comment:
Idk what iota_view is, can you explain?

Reply:
a range type, iterable over the provided values https://en.cppreference.com/w/cpp/ranges/iota_view

Reply:
Its similar to python's `range(0, 10000)`

Reply:
Switched to `std::ranges::views::iota` for clarity.